docs/man/rpm-dependency-generators.7.scd

```diff
@@ -0,0 +1,381 @@
+RPM-DEPENDENCY-GENERATORS(7)
+
+# NAME
+*rpm-dependency-generators* - RPM file attribute and dependency generator
+interface
+
+# SYNOPSIS
+
+## Generators
+*%\_\_*_NAME_*\_conflicts*[*()*] _BODY_++
+*%\_\_*_NAME_*\_enhances*[*()*] _BODY_++
+*%\_\_*_NAME_*\_obsoletes*[*()*] _BODY_++
+*%\_\_*_NAME_*\_orderwithrequires*[*()*] _BODY_++
+*%\_\_*_NAME_*\_provides*[*()*] _BODY_++
+*%\_\_*_NAME_*\_recommends*[*()*] _BODY_++
+*%\_\_*_NAME_*\_requires*[*()*] _BODY_++
+*%\_\_*_NAME_*\_suggests*[*()*] _BODY_++
+*%\_\_*_NAME_*\_supplements*[*()*] _BODY_
+
+## Matching rules
+*%\_\_*_NAME_*\_path* _REGEX_++
+*%\_\_*_NAME_*\_magic* _REGEX_++
```

Comment:
There's also _mime match and the corresponding exclude (_exclude_mime), added in 
e49111d822dc1c0715f5a85ed61523b4d020ae33

Looks like that was never documented although added over five years ago already, oops...

Reply:
Oops :smile: